.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: erlang
+dist: trusty
```

Comment:
Here's the answer: https://github.com/esl/MongooseIM/pull/1133#discussion_r94373464 :)